### DISCUSSION TITLE
Write prompt name into downloaded image

### DISCUSSION THREAD

**Discussion by theridon** [JPhilipp/powerdalle#2] [Ideas]
It would be a great help if there was a way so that when you download a generated image, it defaults to setting the image name as the full prompt. Other image generators like Meta do this (Ironically Dalle via bing does not).  Thanks!

**Comment by JPhilipp** [comment]
Great idea, cheers! Please update to the latest version. It now uses the first 50 characters of your prompt after the datetime in the filename (followed by an id). I hope that's useful.

(It's still using the date as I wanted to keep some easy chronological-sortability of files if you, say, move them around on your disk.)

**Reply by theridon** [reply]
Thanks for the update! When factoring in the need to add more text at the start of the prompt just to get the exact prompt (this is openai's fault)-and adding that with the date at the start-it unfortunately doesn't end up saving much of the actual prompt. It would be awesome if there was an option to remove the date in the future.
